src/libraries/System.Drawing.Common/tests/Drawing2D/LinearGradientBrushTests.cs

```diff
@@ -358,6 +358,17 @@ public void Blend_SetNullBlendFactors_ThrowsNullReferenceException()
         }
 
         [ConditionalFact(Helpers.IsDrawingSupported)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Exception not updated on NetFX")]
+        public void Blend_SetNullBlendPositions_ThrowsArgumentException()
+        {
+            using (var brush = new LinearGradientBrush(new Rectangle(1, 2, 3, 4), Color.Plum, Color.Red, 45, true))
+            {
+                AssertExtensions.Throws<ArgumentException>("value", () => brush.Blend = new Blend { Factors = new float[2], Positions = null });
```

Comment:
These two test cases can be merged into one.  The throws line just changes:

```suggestion
                AssertExtensions.Throws<ArgumentException>(
                    "value",
                    "source",
                     () => brush.Blend = new Blend { Factors = new float[2], Positions = null });
```

(netcore name, netfx name, test code)

Reply:
So all of the cases that got split should be recombined using that overload of Throws

Reply:
Oh didn't know there were such overloads, thanks!